### ISSUE TITLE
Log output doesn't show until close

### ISSUE THREAD

**Issue by NathanW2** [cosmtrek/air#50]
Unsure this is related to my app or air itself but on Windows when I use `log.Print` etc for anything the output doesn't show until I cancel out of the running app and then it will dump everything to stdout.    

Haven't tested another application yet to see if it's me doing something but thought I would just log it here first and close if it's my fault.

**Comment by yanmingsohu** [comment]
On Windows, Add AIR config:
`full_bin = "start /WAIT app.exe"`

This configuration will make the program open and run in a separate window, everything is OK.

Or Run the program in a different directory:
`full_bin = "start /D build /WAIT app.exe"`

**Comment by mohamedmansour** [comment]
@yanmingsohu I tried the same thing on Windows, and log prints don't output anything unless the file is being swapped.  

I refreshed the page N times, then I changed `handler.go`, then all the logs were flushed. This doesn't happen in `go run .` any ideas? In the below logs, it should have said `Received: foo` BEFORE `handlers.go has changed`

```
[start-api] handlers.go has changed
[start-api] building...
[start-api] time="2021-05-22T13:27:56-07:00" level=info msg="Running the sophisticated Gateway at port http://:8080"
[start-api] time="2021-05-22T13:28:00-07:00" level=info msg="Received: foo" prefix=main
[start-api] time="2021-05-22T13:28:00-07:00" level=info msg="Received: foo" prefix=main
[start-api] time="2021-05-22T13:28:00-07:00" level=info msg="Received: foo" prefix=main
[start-api] time="2021-05-22T13:28:01-07:00" level=info msg="Received: foo" prefix=main
[start-api] time="2021-05-22T13:28:02-07:00" level=info msg="Received: foo" prefix=main
[start-api] time="2021-05-22T13:28:03-07:00" level=info msg="Received: foo" prefix=main
[start-api] time="2021-05-22T13:28:03-07:00" level=info msg="Received: foo" prefix=main
[start-api] time="2021-05-22T13:28:53-07:00" level=info msg="Received: foo" prefix=main
[start-api] time="2021-05-22T13:28:53-07:00" level=info msg="Received: foo" prefix=main
[start-api] time="2021-05-22T13:28:54-07:00" level=info msg="Received: foo" prefix=main
[start-api] time="2021-05-22T13:28:54-07:00" level=info msg="Received: foo" prefix=main
[start-api] time="2021-05-22T13:28:54-07:00" level=info msg="Received: foo" prefix=main
[start-api] time="2021-05-22T13:28:54-07:00" level=info msg="Received: foo" prefix=main
[start-api] time="2021-05-22T13:28:55-07:00" level=info msg="Received: foo" prefix=main
[start-api] time="2021-05-22T13:28:55-07:00" level=info msg="Received: foo" prefix=main
[start-api] running...
```

**Comment by Black-Cat** [comment]
Seems to be fixed by pr #214

**Comment by dhruvsaraiya** [comment]
When can I expect new release with this fix?

> I refreshed the page N times, then I changed handler.go, then all the logs were flushed. This doesn't happen in go run . any ideas? In the below logs, it should have said Received: foo BEFORE handlers.go has changed
Facing similar issue in macOs with docker-compose
